scripts/build.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env sh
+
+esbuild_cmd() {
+  format=$1
+
+  outdir="dist/esm"
+  platform="browser"
+
+  if [ "$format" = "cjs" ]; then
+    outdir="dist/cjs"
+    platform="node"
+  fi
+
+  ./node_modules/.bin/esbuild \
+    --outdir=$outdir \
+    --platform=$platform \
+    --target=es2019 \
+    --sourcemap \
+    --bundle \
+    --format=esm \
+    --splitting \
+    --log-level=error \
+    --jsx-factory='React.createElement' \
+    --jsx-fragment='React.Fragment' \
+    --loader:.json=json \
+    --loader:.js=jsx \
+    --external:"react" \
+    --external:"assert" \
+    --external:"crypto" \
+    --external:"http" \
+    --external:"https" \
+    --external:"os" \
+    --external:"stream" \
+    --external:"url" \
+    --external:"util" \
+    src/index.js
+
+  if [ "$format" = "esm" ]; then
+    fix_esm_default "@walletconnect/web3-provider/dist/cjs/index.js"
+    fix_esm_default "walletlink/dist/index.js"
+  fi
+}
+
+fix_esm_default() {
+
+  # This is an awful solution to a problem that couldn’t get solved in any
+  # other way (as of 2020-08-07). WalletConnect uses CommonJS with
+  # the `__esModule: true` property, indicating that the CJS module can be
+  # transformed into ESM. Unfortunately, it ends up being exported
+  # as `export default { default: value }` rather than `export default value`,
+  # in several bundlers.
+
+  # In the end, the decision was made to:
+  # - Use esbuild.
+  # - Remove SquareLink.
+  # - Fix the WalletConnect module using a search & replace.
```

Comment:
Is there anything WalletConnect could do to help fix this? I'm sure @pedrouid would love to hear it!

Reply:
Provide ESM exports would be amazing :sweat_smile: